src/containers/apply/SelectedApis.tsx

```diff
@@ -67,26 +71,80 @@ const apiInfo = [
   },
 ];
 
-const SelectedAPIs = (): JSX.Element => (
-  <fieldset className="vads-u-margin-top--3">
-    <legend className={classNames('vads-u-font-weight--normal', 'vads-u-font-size--base')}>
-      Please select all of the APIs you&apos;d like access to:
-    </legend>
-    <fieldset
-      className="vads-u-margin-top--2"
-      aria-label="Please select all of the Standard APIs you'd like access to:"
-    >
-      <legend className="vads-u-font-size--lg">Standard APIs:</legend>
-      <ApiCheckboxList apiCheckboxes={apiInfo} />
-    </fieldset>
+interface SelectedApisProps {
+  selectedApis: string[];
+}
+
+const SelectedAPIs = ({ selectedApis }: SelectedApisProps): JSX.Element => {
+  const { errors } = useFormikContext();
+  const checkboxName = 'apis';
+  const shouldDisplayErrors = !!errors[checkboxName];
+  const containerClass = shouldDisplayErrors ? 'usa-input-error' : '';
+  const labelClass = shouldDisplayErrors ? 'usa-input-error-label' : '';
+  const validationClass = shouldDisplayErrors ? 'usa-input-error-message' : '';
+  const errorMessagePaddingClass = shouldDisplayErrors ? 'vads-u-padding-x--1p5' : '';
+  const selectAPIClass = shouldDisplayErrors
+    ? 'vads-u-font-weight--bold'
+    : 'vads-u-font-weight--normal';
+
+  const oauthApisSelected = anyOAuthApisSelected(selectedApis);
+  const oauthApisBorderClass = oauthApisSelected ? 'vads-u-border-left--4px' : '';
+  const oauthApisBorderColorClass = oauthApisSelected
+    ? 'vads-u-border-color--primary-alt-light'
+    : '';
+
+  return (
     <fieldset
-      className="vads-u-margin-top--2"
-      aria-label="Please select all the OAuth APIs you'd like access to:"
+      aria-labelledby="select-checkbox-api"
+      className={classNames(containerClass, 'apply-api-select')}
```

Comment:
Was there a reason for dropping the utility class? I think we should keep them if possible to avoid duplication.
```suggestion
      className={classNames(
        containerClass,
        'vads-u-margin-top--2p5',
        'vads-u-background-color--gray-lightest',
      )}
```

Reply:
done